src/main/java/dev/jbang/util/Util.java

```diff
@@ -1695,16 +1696,16 @@ public static Path searchPath(String cmd, String paths, Predicate<Path> pathFilt
 
 	private static Stream<Path> executables(Path base) {
 		if (isWindows()) {
-			return Stream.of(Paths.get(base.toString() + ".exe"),
-					Paths.get(base.toString() + ".bat"),
-					Paths.get(base.toString() + ".cmd"),
-					Paths.get(base.toString() + ".ps1"));
+			return Stream.of(Paths.get(base + ".exe"),
```

Comment:
Will this work if in a bash env on windows that we don't check base ?

Reply:
This code has been here for ages, it has not changed. I only removed the unneeded `.toString()`.

But on WIndows things like bash will still run .exe and .bats. Given the fact that we only use `searchPath()` for finding executables and not bash scripts we're okay not including the extension-less name on Windows. Also, in the cases we'd like to find a command that might be a bash script, eg. `jbang` itself. On Windows it still preferable to run the .cmd or .ps1 version. So unless we have a situation where we positively want to find a bash script on Windows this code will mostly do the right thing.